Trigger/src/PrescaleEvent_module.cc

```diff
@@ -91,7 +91,7 @@ namespace mu2e
     bool retval(false);
     bool condition = e.event() % ps == 0;
 
```

Comment:
Separate comment on the preceding line.  Have you thought about  using ( _nevt % ps ) instead of (e.event() % ps).  When we are running in the trigger farm there will be short term structure in the event numbers seen by any given trigger process. I don't have a good quantitative picture of what short term is.   I think in the long term event builders that pull events from off-machine board readers will introduce enough randomness.  The issue is the short term - could we get all events that pass landing on a small number of trigger processes.

This situation will never come up with MC as we structure it today.

Reply:
I didn't think about this TBH. Probably we can figure this out when we will do a larger scale test later on.